demisto_sdk/commands/common/tools.py

```diff
@@ -539,16 +540,19 @@ def get_remote_file_from_api(
     file_content = res.content
     if return_content:
         return file_content
-    return get_file_details(file_content, full_file_path)
+    return get_file_details(file_content, full_file_path, keep_order=True)
 
 
 def get_file_details(
     file_content,
     full_file_path: str,
+    keep_order: bool = False,
 ) -> Dict:
     if full_file_path.endswith("json"):
         file_details = json.loads(file_content)
     elif full_file_path.endswith(("yml", "yaml")):
+        if keep_order:
+            file_details = yaml_ordered_load.load(file_content)
         file_details = yaml.load(file_content)
```

Comment:
```suggestion
        file_details = yaml_ordered_load.load(file_content) if keep_order else yaml.load(file_content)
```

Reply:
or, the chaotic equivalent (don't)

```suggestion
        file_details = (yaml_ordered_load if keep_order else yaml).load(file_content)
```